src/ImportDefinitionsBundle/Exporter/Exporter.php

```diff
@@ -160,9 +161,15 @@ private function runExport(ExportDefinitionInterface $definition, $params, int $
         $countToClean = 1000;
         $perLoop = 50;
 
+        $fetcherConfig = is_array($definition->getFetcherConfig()) ? $definition->getFetcherConfig() : [];
```

Comment:
I am not sure if thats actually the right position for that. The fetcher config shouldn't be used here. We could introduce a new boolean config directly into the `ExportDefinition` entity and configure that from there.

Reply:
I thought about this too, agree from the technical POV, but not from the UX POV.

If you think about it, you're configuring what objects will be pulled in. By saying "Fetch unpublished", you fetch both the objects from the listing and from all relations. If not, you don't (in both cases).

Having this be separate will probably be confusing since understanding it requires knowing how Pimcore works, which is not the baseline for your UX, AFAIK?

Reply:
What about using the `ExportDefinition` flag for both and removing the `unpublished` from the fetcher?

Reply:
That works too, as long as the semantics is the same: one flag to manage both (unfortunately separate by underlying system, for some reason) subsystems. :+1: